### PR TITLE
Request for Akzana.is-a.Dev subdomain

### PR DESCRIPTION
**Pull request by akzana**
# About this Patch
+ I want Akzana.is-a.dev as a subdomain for my professional portfolio.
+ I will host project previews on this site for employers and clients to view my skillset.

## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [X] The file is in the `domains` folder and is in the JSON format.
- [X] The file's name is all lowercased and alphanumeric. 
- [X] You have completed your website. 
- [X] The website is reachable.  
- [X] The CNAME record doesn't contain `https://` or `/`.  
- [X] There is sufficient information at the `owner` field.  

## Website Link/Preview

[akzana.github.io](https://akzana.github.io/)

### CODE REVIEW

File: domains/akzana.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "description": "Portfolio Site for Akzana. I want to host small projects on Akzana.Is-A.Dev to showcase my software engineering skills.",
+  "repo": "https://github.com/akzana/akzana.github.io",
+  "owner": {
+    "username": "akzana",
+    "email": "akzanaklepsch@outlook.com"
+  },
+  "record": {
+    "CNAME": "akzana.github.io",
+    "TXT": ["a9e7df1f4744431d00299a1ca92510"]
```
Comment (Stef-00012) [inline domains/akzana.json:10]: 1. You can't use CNAME with other records
2. 2. is that TXT a github pages verification string? if so you have to put it ina  different file, see https://is-a.dev/docs/github-pages-verification

**Review by Stef-DP — DISMISSED** [review]

**Comment by akzana** [comment]
OK I will make the changes and update this PR

File: domains/akzana.json (hunk 1 above)
Comment (akzana) [reply domains/akzana.json:10]: Thank you for the information. I will update this PR with the Github verification

Comment (akzana) [reply domains/akzana.json]: Hello again, 
I've updated my request with another JSON file with the github pages verification. Let me know if there's still wrong info.

**Comment by akzana** [comment]
Thanks Maintainers! Glad to be a part of the community

**Pull request merged**